src/api/services/NotificationAPI.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * NotificationAPI Services
+ * Provide services related to Notification (Avvisatura) to communicate with PagoPaAPI
+ */
+
+import * as express from "express";
+import fetch from "node-fetch";
+import { CONFIG } from "../../Configuration";
+import { NotificationSubscriptionRequestType } from "../../enums/NotificationSubscriptionType";
+import { NotificationSubscriptionResponse } from "../types/NotificationSubscriptionResponse";
+
+// Notification Service for PagoPA communications
+export class NotificationAPI {
+  // Update subscription (Activation or Deactivation) to Notification Service for a fiscalCode
+  public static updateSubscription(
+    fiscalCode: string,
+    requestType: NotificationSubscriptionRequestType,
+    res: express.Response,
+    errorCallback: (res: express.Response, errorMsg: string) => void,
+    successCallback: (
+      res: express.Response,
+      notificationSubscriptionResponse: NotificationSubscriptionResponse
+    ) => void
+  ): void {
+    const url =
+      CONFIG.PAGOPA.HOST +
+      ":" +
+      String(CONFIG.PAGOPA.PORT) +
+      CONFIG.PAGOPA.SERVICES.NOTIFICATION_UPDATE_SUBSCRIPTION;
+    const body = {
+      timestamp: Date.now(),
+      requestId: Math.floor(Math.random() * Number.MAX_SAFE_INTEGER),
```

Comment:
I'd use uuids

Reply:
ok, I'm using v1 from:
https://www.npmjs.com/package/uuid
It' based to timestamp, so I think it's the best version to use.
v4 is just random, It could create collisions for my scope
@cloudify do you agree?

Reply:
yes, but please capture this decision in an [ADR](https://github.com/teamdigitale/italia-pagopa-proxy/pull/25) like we did [here](https://github.com/teamdigitale/italia-backend/blob/master/doc/architecture/decisions/0005-use-a-guid-as-installation-id.md)